.github/workflows/test.yml

```
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install modules
+        run: yarn
+      - name: Run tests
```

- There should be a line break between each step
- The steps should have names
- We should cache the node modules to reduce the build time - ref: https://github.com/nimblehq/elixir-templates/blob/ee2d57280db231432adb4b4cd96bcfcfcac79d79/priv/templates/nimble_template/.github/workflows/test.yml.eex#L50-L56

```suggestion
      - name: Checkout the repository 
        uses: actions/checkout@v2
        
      - name: Setup Node
        uses: actions/setup-node@v2
        with:
          node-version: ${{ matrix.node-version }}
      
      - name: Cache Node npm
        uses: actions/cache@v2
        with:
          path: assets/node_modules
          key: ${{ runner.os }}-zipworld-node-${{ hashFiles('**/package-lock.json') }}
          restore-keys: |
            ${{ runner.os }}-zipworld-node-
              
      - name: Install Node modules
        run: yarn

      - name: Run tests
```

---

Updated the workflow: https://github.com/nimblehq/infrastructure-templates/pull/22/commits/86f86a8ab97b5db23d910d244eba53cb44f96a88